src/thank-you.hbs

```diff
@@ -0,0 +1,18 @@
+---
+title: Thank you!
+description: Thank you for supporting our continued work to develop products for student success
```

Comment:
No need to add a description here since we're not indexing it, right?

Reply:
True 😄